diffkemp/semdiff/result.py

```diff
@@ -259,6 +269,10 @@ def __hash__(self):
             print("Elapsed time:            {:.2f} s".format(
                 self.stop_time - self.start_time))
         print("Functions compared:      {}".format(compared))
+        print("Lines compared:          {}".format(compared_lines))
+        print("Instructions compared:   {}".format(compared_instructions))
+        print("1:1 equal instructions:  {0} ({1:.0f}%)".format(
+            equal_instructions, equal_percent))
```

Comment:
Let's insert one blank line here (before `Total differences:`) to visually separate the following block.